### PR TITLE
feature/link-external-resources

### PR DESCRIPTION
**Pull request by roja95**
Add links to useful resources to the bottom.

### CODE REVIEW

**Review by vlado — COMMENTED** [review]
Looks good, can you wrap it with "div.content" https://bulma.io/documentation/elements/content/?

That should make sure that Bulma handles the styling I think.

**Comment by roja95** [comment]
sure

**Comment by roja95** [comment]
Done, should I make some more changes or is this good enough?

**Comment by vlado** [comment]
> Done, should I make some more changes or is this good enough?

Can you put a screenshot?

**Comment by roja95** [comment]
image

**Comment by roja95** [comment]
image

File: app/views/ads/index.html.erb [hunk 1]
```diff
@@ -103,3 +103,11 @@
 <% end %>
 
 <%= will_paginate @ads, renderer: BulmaPagination::Rails, class: 'is-centered', previous_label: "&larr;", next_label: "&rarr;" %>
+
+<div class="content">
+  <ul>Ostale stranice sa popisima ponude smještaja i prijevoza:
```
Comment (vlado) [inline app/views/ads/index.html.erb:108]: Sad sam skužio jednu grešku.

```suggestion
  <h3 class="title is-3">Ostale stranice sa popisima ponude smještaja i prijevoza:</h3>
  <ul>
```

Comment (vlado) [reply app/views/ads/index.html.erb:108]: Inače super izgleda 👍

Comment (roja95) [reply app/views/ads/index.html.erb]: Istina tvoja verzija je bolja. Hvala commito sam suggestion pa se sad valjda moze mergat.

**Review by vlado — APPROVED** [review]

**Comment by vlado** [comment]
@roja95 Very nice 🍰 

Thanks!

**Pull request merged**